@lohedges I am not sure who commented out that test. When I tested it a few weeks ago on Windows it was still running after one day. On my Linux Python3 build it fails with a weird backtrace:
```
Traceback (most recent call last):
  File "test_locale.py", line 13, in <module>
    from nose.tools import assert_equal
  File "/build/common/third-party/Sire/bin/trunk/linux-x86_64/lib/python3.6/site-packages/nose/__init__.py", line 1, in <module>
    from nose.core import collector, main, run, run_exit, runmodule
  File "/build/common/third-party/Sire/bin/trunk/linux-x86_64/lib/python3.6/site-packages/nose/core.py", line 11, in <module>
    from nose.config import Config, all_config_files
  File "/build/common/third-party/Sire/bin/trunk/linux-x86_64/lib/python3.6/site-packages/nose/config.py", line 9, in <module>
    from nose.plugins.manager import NoPlugins
  File "/build/common/third-party/Sire/bin/trunk/linux-x86_64/lib/python3.6/site-packages/nose/plugins/__init__.py", line 185, in <module>
    from nose.plugins.manager import *
  File "/build/common/third-party/Sire/bin/trunk/linux-x86_64/lib/python3.6/site-packages/nose/plugins/manager.py", line 418, in <module>
    import pkg_resources
  File "/build/common/third-party/Python/bin/trunk/linux-x86_64/lib/python3.6/site-packages/pkg_resources/__init__.py", line 26, in <module>
    import zipfile
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/build/common/third-party/Python/bin/trunk/linux-x86_64/lib/python36.zip/zipfile.py", line 13, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/build/common/third-party/Python/bin/trunk/linux-x86_64/lib/python36.zip/shutil.py", line 29, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/build/common/third-party/Python/bin/trunk/linux-x86_64/lib/python36.zip/lzma.py", line 27, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 54: invalid continuation byte
```
It works fine if I do not import the `nose.tools.assert_equal`. Note that all other `nose` tests work OK - I think it has to do with the fact that the test calls itself.

--- a/unittests/SireIO/test_locale.py
+++ b/unittests/SireIO/test_locale.py
@@ -10,9 +10,10 @@ import shlex
 import sys
 import os
 
-#from nose.tools import assert_equal
+from nose.tools import assert_equal
 
-sire_python = sys.executable
+sire_python = os.path.join(os.path.dirname(sys.executable), "sire_python")
+
 gromacs_path = StringProperty("../io/gromacs")
 
 def _test_broken_pdb():
@@ -42,14 +43,14 @@ def _test_broken_pdb():
 
     m2 = s[MolIdx(0)]
 
-    assert( m.nAtoms() == m2.nAtoms() )
+    assert_equal( m.nAtoms(), m2.nAtoms() )
 
     for i in range(0, m.nAtoms()):
         if verbose:
             print("%s vs %s" % (m.atoms()[i].property("coordinates"),
                                 m2.atoms()[i].property("coordinates")))
 
-        assert( m.atoms()[i].property("coordinates") == 
+        assert_equal( m.atoms()[i].property("coordinates"), 
                       m2.atoms()[i].property("coordinates") )
 
     # now try the same with PDB2
@@ -66,14 +67,14 @@ def _test_broken_pdb():
 
     m2 = s[MolIdx(0)]
 
-    assert( m.nAtoms() == m2.nAtoms() )
+    assert_equal( m.nAtoms(), m2.nAtoms() )
 
     for i in range(0, m.nAtoms()):
         if verbose:
             print("%s vs %s" % (m.atoms()[i].property("coordinates"),
                                 m2.atoms()[i].property("coordinates")))
 
-        assert( m.atoms()[i].property("coordinates") ==
+        assert_equal( m.atoms()[i].property("coordinates"),
                       m2.atoms()[i].property("coordinates") )
 
 def _test_broken_rst7():
@@ -110,7 +111,7 @@ def _test_broken_rst7():
             v1 = m1.atoms()[j].property("coordinates")
             v2 = m2.atoms()[j].property("coordinates")
 
-            assert(v1 == v2)
+            assert_equal(v1, v2)
 
     if verbose:
         print("All ok :-)")    
@@ -150,7 +151,7 @@ def _test_broken_gro():
             v1 = m1.atoms()[j].property("coordinates")
             v2 = m2.atoms()[j].property("coordinates")
 
-            assert(v1 == v2)
+            assert_equal(v1, v2)
 
     if verbose:
         print("All ok :-)")    
@@ -162,7 +163,7 @@ def _test_broken_function(verbose, function):
     if verbose:
         print(cmd)
 
-    env = dict(os.environ, LC_ALL="fr_FR")
+    env = dict(os.environ, LC_ALL="it_IT")
 
     if verbose:
         env["VERBOSE_TEST"] = "1"

